src/dataframe/dataframe.jl

```diff
@@ -698,7 +747,16 @@ end
 ##############################################################################
 
 # hcat! for 2 arguments, only a vector or a data frame is allowed
-function hcat!(df1::DataFrame, df2::AbstractDataFrame)
+function hcat!(df1::DataFrame, df2::AbstractDataFrame; make_unique::Bool=false)
+    common = intersect(_names(df1), _names(df2))
+    if !make_unique && length(common) > 0
```

Comment:
Shouldn't `make_unique` be passed when inserting columns? Else a warning will be printed downstream?

Reply:
changed. now it will as `add_names` is downstream and I have to make it accept `makeunique` anyway given other comments.